libs/ui/src/lib/primary-button/primary-button.component.ts

```diff
@@ -46,6 +57,10 @@ export class PrimaryButtonComponent {
   @Input() routerLink!: string;
   @Input() type!: string;
   @Input() disabled = false;
+  @Input() isLoading = false;
+  @Input() lodingMessage = 'Laden...';
```

Comment:
```suggestion
  @Input() loadingMessage = 'Laden...';
```
@JulianTurner Seiteneffekte dieser Ännderung?